public/index.html

```diff
@@ -25,7 +25,12 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <!-- <script data-ad-client="ca-pub-7058184148167069" async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script> -->
-    <title>Json Prettier</title>
+    <title>Json Prettier - The best and securely opensource json formatter!</title>
+    <meta name="google-site-verification" content="kIn34s4kN-pinOiHQ_5FUThf0fmmdSEAefjdXen7CYw" />
```

Comment:
Verify your website, so you can use google console to analyze your website metrics: https://support.google.com/webmasters/answer/9008080

After you create an account there, I could update this code here.